Command/GifOptimizerCommand.php

```diff
@@ -24,17 +25,17 @@ class GifOptimizerCommand extends Command
     /**
      * @var string The name of the command
      */
-    const COMMAND_NAME = 'jolicode:gifexception:optimize';
+    public const COMMAND_NAME = 'jolicode:gifexception:optimize';
```

Comment:
All const should be private.
In the test, the command name should be hardcoded
(If one change the value here, test will be OK, it should not because it's a BC break)